### ISSUE TITLE
Requirements

### ISSUE THREAD

**Issue by arduino43** [tanisman/heliumover#2]
Excellent project.

I have the admin panel running, I am wondering if this requires the miner to have the service installed or if this can act as a packet forwarder without modification to miner?

Thanks!

**Comment by tanisman** [comment]
you must install /proxy folder to your miner there is a dockerfile that you can build for miner. To install on browan and pantherx simple modify lora_pkt_fwd_start.sh and copy it to /usr/bin and restart your miner.

Installing on sensecap is a bit tricky, you must first build docker image in /pkt_fwd_sensecap  and than create systemd service (check /proxy/heliumover.service).

all docker images should build for arm v7

**Comment by arduino43** [comment]
Thank you for the quick reply.

I was a bit confused as to what exactly is happening - you're installing a service on a miner to forward packets to a proxy which then modifies the packets and sends it to other groups of miners.

I didn't follow why you would even need to modify any of the packet forwarder configurations, but perhaps I am missing something. You can just create a semtech middleman and forward the packets to port 1680 and the miner will accept these packets.

**Comment by tanisman** [comment]
you are right, modified packets could have directly redirected to 1680 but static IP is a must for that since getting API responses from helium API is stupidly slow and highly rate-limited, plus initial version of this project was using another technique via app installed on miner using MQTT to share packets and each miner was locally calculating their RSSI and SNR. This is modified version of the previous version so using old code was pretty useful. Besides that, I also wanted to have full control of the packet forwarded to overcome any future attempts to prevent cheating on miner software by helium team.

**Comment by arduino43** [comment]
Yes all of this make sense.

By any chance do you have a way to contact? if you could email me at planers-marmot0y AT icloud.com I would like to send something.
